cv/style.css

```diff
@@ -0,0 +1,203 @@
+body {
+  font-family: "Roboto", sans-serif;
+  color: #545E6C;
+  background: #f5f5f5;
+  font-size: 14px;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-weight: 700;
+}
+
+a {
+  color: #2d7788;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+  color: #1a454f;
+}
+
+a:focus {
+  text-decoration: none;
+}
+
+p {
+  line-height: 1.5;
+}
+
+.wrapper {
+  display: flex;
+  max-width: 960px;
+  margin: 0 auto;
```

Comment:
You should stick to using flexbox for positioning. To achieve the same result you can do 
```
    justify-content: center;
    display: flex;
 ```
 on `body`
 and remove the `  margin: 0 auto;`
 
 
 or you could've create another div between body and wrapper with width 100% and give it 
 ```
    justify-content: center;
    display: flex;
    ```